cmd/package.go

```diff
@@ -316,7 +305,36 @@ func CollectPackage(repo *util.Repo, packageDir string, pullMissing bool, custom
 			}
 		}
 
-		return nil
+		// Ignore what needs to be ignored.
+		if capstanignore.IsIgnored(relPath) {
+			if verbose {
+				display := true
+				for _, pattern := range core.CAPSTANIGNORE_ALWAYS {
```

Comment:
Any particular reason for this? I can imagine that for `.git` without this, it will print all files within `.git` dir which is far from useful. However, if you simply return `SkipDir` when you come to a directory that is ignored? This will improve performance on one hand. On the other, you don't need this part because you can always print the verbose message when a file/dir is ignored.